### PR TITLE
Fix waypoint description to be in accordance with W3C rec

### LINKED ISSUE #1127: "click here", realurlasexample.com
http://www.freecodecamp.com/challenges/waypoint-wrap-an-anchor-element-within-a-paragraph tells us to write "click here" in front of a link, which according to http://www.w3.org/QA/Tips/noClickHere is problematic because "click" is a mechanic and not descriptive. Instead the text to be added could be "View more "

Also real-world URLs should probably not be used. Perhaps example.com/catapp instead of catphotoapp.com?

### PR DESCRIPTION
**Pull request by butterthebuddha**
Closes #1127

### CODE REVIEW

**Comment by bugron** [comment]
@awesomeaniruddh you, actually, don't check if the text of `p` element is 'View more', you've changed only the message.

File: seed/challenges/html5-and-css.json [hunk 1]
```diff
@@ -1133,14 +1133,14 @@
...
       ],
       "tests": [
         "assert($(\"a\").attr(\"href\").match(/freecatphotoapp.com/gi).length > 0, 'You need an <code>a</code> element that links to \"freecatphotoapp.com\".')",
         "assert($(\"a\").text().match(/cat\\sphotos/gi), 'Your <code>a</code> element should have the anchor text of \"cat photos\"')",
         "assert($(\"p\") && $(\"p\").length > 2, 'Create a new <code>p</code> element around your <code>a</code> element.')",
         "assert($(\"a[href=\\\"http://www.freecatphotoapp.com\\\"]\").parent().is(\"p\"), 'Your <code>a</code> element should be nested within your new <code>p</code> element.')",
-        "assert($(\"p\").text().match(/click\\shere\\sfor/gi), 'Your <code>p</code> element should have the text \"click here for\".')",
+        "assert($(\"p\").text().match(/click\\shere\\sfor/gi), 'Your <code>p</code> element should have the text \"View more\".')",
         "assert(editor.match(/<\\/p>/g) && editor.match(/<p/g) && editor.match(/<\\/p>/g).length === editor.match(/<p/g).length, 'Make sure each of your <code>p</code> elements has a closing tag.')",
```
Comment (BerkeleyTrue) [inline seed/challenges/html5-and-css.json]: @bugron Is correct, the test is still checking for the old copy.

**Comment by awesomeaniruddh** [comment]
Oh I forgot to push the amended commit

File: seed/challenges/html5-and-css.json (hunk 1 above)
Comment (QuincyLarson) [inline seed/challenges/html5-and-css.json]: @BerkeleyTrue is correct - please update the test as well and verify that it works locally. Then we will merge this.

**Comment by awesomeaniruddh** [comment]
That should be it :)

**Comment by BerkeleyTrue** [comment]
@awesomeaniruddh Thanks for you help. Happy Coding.

**Pull request merged**